.github/lock.yml

```diff
@@ -0,0 +1 @@
+daysUntilLock: 30
```

Comment:
Probably worth adding a comment too `lockComment`

Reply:
I don't have any ideas for improving over the default though:  "This thread has been automatically locked because it has not had recent activity. Please open a new issue for related bugs and link to relevant comments in this thread."

Reply:
oh, i missed that that was the default.
The only other comment i have is that the lock timeout should be longer (i was gonna do 60 days), because it could be almost 30 days between a fix being merged and being released.

Reply:
does closing the ticket reset the timer?

Reply:
(i assumed it did, IDK)

Reply:
locking an issue definitely counts as an update.  e.g. https://github.com/chef/chef/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed++updated%3A%3E%3D2018-04-02 find https://github.com/chef/chef/issues/6606 because i locked that on the 2nd.

Reply:
![screen shot 2018-04-09 at 3 54 08 pm](https://user-images.githubusercontent.com/454857/38527194-605c2922-3c0e-11e8-89d9-e987a585a0e1.png)

yeah, the timer resets when an issue is closed -- so this is 30 days from when it was closed/fixed at a minimum.

Reply:
yeah, which might be only a day or two after we do a stable release with the fix